InventoryManagementSystem/Seeder/RepositorySeeder.cs

```diff
@@ -0,0 +1,22 @@
+﻿using InventoryManagementSystem.DB;
+using InventoryManagementSystem.Models;
+
+namespace InventoryManagementSystem.Inventory
+{
+    internal class RepositorySeeder
+    {
+        private readonly IRepository _repository;
+
+        public RepositorySeeder(IRepository repository)
+        {
+            _repository = repository;
+        }
+
+        internal void SeedInventory()
+        {
+            _repository.AddProduct(new Product { Name = "table", Price = 60, Quantity = 164 });
```

Comment:
you need to make sure the seeding data does not exist before adding it. since you're using a database and you're gonna run the seeding code each time the application start.